src/canopy/utils/config.py

```diff
@@ -1,6 +1,18 @@
+import os
+import yaml
 from typing import Dict, Any
 
 
+def load_config_template(name: str) -> Dict[str, Any]:
+    if not name.endswith(".yaml"):
+        name += ".yaml"
+
+    config_template_path = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                                        "config_templates", name)
+    with open(config_template_path, "r") as f:
+        return yaml.safe_load(f)
```

Comment:
Maybe have a Directory class in the init file to refer to specific directories with the project. Sth like:

class Directory:
    Root = Path(....)
    ConfigTemplates = self.Root.joinpath(...)